net-misc/curl/metadata.xml

```diff
@@ -36,8 +36,6 @@
 		<flag name="zstd">Enable zstd compression</flag>
 	</use>
 	<upstream>
-		<remote-id type="cpe">cpe:/a:curl:curl</remote-id>
-		<remote-id type="cpe">cpe:/a:curl:libcurl</remote-id>
 		<remote-id type="cpe">cpe:/a:haxx:curl</remote-id>
```

Comment:
I'm a bit worried about this because Daniel is actively trying to move away from the _haxxe_ name and scanners (as you noted in your bug) should really go for any-of.

Reply:
I see, I don't know much about changing a CPE for a package but I imagine it's quite complicated. First all of the history of the other CPE will have to be transferred/backfilled to ensure you don't lose information/ have to adjust logic to include using different cpe's depending on the version. Then anything depending on this would have to be alerted to change and without some sort of transition period it's likely many consumers will go unpatched for a long time. So I guess with this I have a couple questions. Do you know if he is planning to move to the cpe's listed there? Cause if not, there is still is no point in having them there. Also, do you know how long a process like this typically takes? I've never seen it happen.